### DISCUSSION TITLE
Github Classroom

### DISCUSSION THREAD

**Discussion by ShantiRoseCodes** [community/community#56442] [GitHub Education]
### Select Topic Area

Question

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

Students can’t access their assignment as they receive a message that says they have exceeded the number of repos.
![IMG_2008](https://github.com/community/community/assets/87835294/f4829528-883c-41bf-b0a5-2e89915fb887)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ShantiRoseCodes ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by markpatterson27** [comment]
It looks like a limit on the organisation that the GitHub Classroom is creating assignments in. It's not the students exceeding their number of repos, it's the organisation owner.

How many students do you have in the class and how many Classroom assignments do they need to accept?

Has the organisation been upgraded? By the looks of the error message, it hasn't. You might get a larger quota of private repos if you do. You can request an upgrade by using the green 'Upgrade to GitHub Team' button on the page at [https://education.github.com/globalcampus/teacher](https://education.github.com/globalcampus/teacher).

You could consider using more than one organisation for all the students' assignments. Orgs are free to create :). It's usually best to create a seperate org for each Classroom (Unfortunately, a Classroom can only be linked to one org).

Are you re-using the same org for each year? Do you need to keep previous years' assignments? Another thing to consider here, if you are in the EU, keeping assignment repos for longer than necessary could be considered a breach of GDPR. If you need to go through and delete old repos, I would definately script the process. The GitHub API isn't that difficult to use (@ Message me back if you need any help with it).
